LoopKit/Extensions/NSUserActivity+CarbKit.swift

```diff
@@ -13,17 +13,19 @@ extension NSUserActivity {
     public static let newCarbEntryActivityType = "NewCarbEntry"
 
     public static let newCarbEntryUserInfoKey = "NewCarbEntry"
+    public static let carbEntryIsUAMUserInfoKey = "CarbEntryIsUAM"
```

Comment:
Hmm, this class looks like it should probably move to Loop, now that the carb entry interface lives there. Not something you need to do for this ticket; not really related; I just noticed.

Reply:
I can mark it as a FIXME